packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md

```diff
@@ -26,15 +27,62 @@ Customize error pages as part of the sign-in flow.
+The [Brands API](/docs/reference/api/brands/) allows you to set icons, images, and colors across the following:
+
+- sign-in page (redirect authentication)
+- error pages
+- email templates
+- Okta End-User Dashboard
+
+You don't need to set a customized Okta URL domain. See [Customize your Okta experience with the Brands API](/docs/guides/customize-themes) for more information on this feature and how to use it.
+
+## Content Security Policy (CSP) for your custom domain
+
+Set up a [custom domain](/docs/guides/custom-url-domain/main/) and customize your [CSP](https://content-security-policy.com/) if you also want to customize the sign-in page or error pages. CSP customizations only take effect on custom domains.
+
+To analyze and detect potentially malicious IP addresses that seek to bypass your CSP, use [Okta ThreatInsight](https://help.okta.com/okta_help.htm?type=oie&id=ext-about-threatinsight).
+
+> **Note:** If you add too many trusted origins to your custom CSP, the HTTP header size can exceed the limit allowed by some server software. Update the default server limits or reduce the number of trusted origins.
+
+> **Note:** You can have a maximum of 20 URIs.
+
+> **Note:** If you add a trusted origin URL that redirects to a different URL, you need to include the redirect URL on the trusted origin list.
+
+### Add a nonce reference
+
+Add a [`nonce`](https://content-security-policy.com/nonce/) reference to your HTML if you want to customize the Okta-hosted sign-in page or error pages. Without the `nonce` reference, your `script` and `style` tags don't run.
+
+To add the `nonce` reference, include it as a variable in the code editor. See [Edit the error page](#edit-the-error-page).
+
+Example:
+
+```html
+<style nonce="{{nonceValue}}">
+```
+
+### Customize the default CSP
+
+1. In the Admin Console, go to **Customizations** > **Branding**.
+2. In the **Pages** panel, click **Edit** for **Error pages**.
+3. Click **Settings**.
+4. In the **Content Security Policy** panel, click **Edit**. Set the following:
+
+   - **Trusted external resources**: Add resources to the CSP. For example: mydomain.com, *.mydomain.com, or mydomain.com/images. Click **Add**.
+     > **Note:** Okta adds these resources to all fetch-directives in the CSP.
+   - **Report UI**: Enter the URI to which you want to send violation report details. The URI entered here appears in the report-uri directive of the CSP.
+   - **Enforcement**:
+     - Select **Enforced** to block any resources that the CSP doesn't trust. The header is `content-security-policy`.
+     - Use **Not enforced** to leave the customized CSP in the report-only header (`content-security-policy-report-only`). Use this option for testing and validation before turning on **Enforced** mode.
+5. Click **Save to draft**.
+6. Make changes directly in the editor. See steps 4 and 5 in [Edit the error page](#edit-the-error-page).
```

Comment:
Maybe "Make your changes in the editor."? Not sure what directly adds.

Reply:
done